tools/pks.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Install pivnet cli
+source `dirname "${BASH_SOURCE[0]}"`/../install.sh pivnet
```

Comment:
what about `./` instead?

Reply:
Good point, for install creating a sub shell make more sense as it's not intended to alter the environment of the current shell.

I think it's ok for this line to be a source, but inside the install script should be a `./`

Done.